services/src/main/java/org/fao/geonet/api/processing/report/InfoReport.java

```diff
@@ -23,15 +23,9 @@
 
 package org.fao.geonet.api.processing.report;
 
-import org.opengis.annotation.XmlElement;
-
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 
-/**
- * Created by francois on 20/05/16.
- */
-@XmlElement(value = "metadata")
```

Comment:
@fxprunayre I tested the report and looks fine, but I'm not sure why this is removed?

Reply:
See above, apparently XmlElement annotation not applicable to a class (kind of makes sense as a class should match to a type)